engine/src/main/resources/application.conf

```diff
@@ -36,6 +36,9 @@ system {
 
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
   abort-jobs-on-terminate = false
+
+  // Max number of retries per job that the engine will attempt in case of a retryable failure received from the backend
+  maxRetries = 3
```

Comment:
While this feature is obviously supported on the engine side, I agree with @Horneth that this should be specified within the backend and not at the engine level.

Also a nitpick that wherever this is implemented, this file is generally moving over to `lower-case-separated-by-dashes` for key names.